src/main/java/model/LottoGenerater.java

```diff
@@ -0,0 +1,22 @@
+package model;
+
+import static constant.LottoConstant.LOTTO_NUMBER_COUNT;
+
+import java.util.Set;
+import java.util.TreeSet;
+
+public class LottoGenerater {
+    private final LottoNumberPicker lottoNumberPicker;
+
+    public LottoGenerater(LottoNumberPicker lottoNumberPicker) {
+        this.lottoNumberPicker = lottoNumberPicker;
+    }
+
+    public Lotto generateLotto() {
+        Set<Number> numbers = new TreeSet<>();
+        while(numbers.size() != LOTTO_NUMBER_COUNT) {
```

Comment:
```suggestion
        while (numbers.size() != LOTTO_NUMBER_COUNT) {
```
코드 전반적으로 [Google Style Guides](https://google.github.io/styleguide/javaguide.html)를 지키고 있는지 체크해볼까요?